app/views/pages/contact.html.erb

```diff
@@ -1,6 +1,28 @@
 <h1 class="govuk-heading-l govuk-!-margin-bottom-5">Contact</h1>
 
 <div class="govuk-grid-row dashboard-row">
+
   <div class="govuk-grid-column-two-thirds">
+    <%= form_tag(contact_path, method: :post, id: "contact_form") do %>
+      <div class="govuk-form-group">
+        <label class="govuk-label" for="more-detail">
+          You can use this form to ask a question, report a problem or suggest an improvement.
+        </label>
+        <span id="more-detail-hint" class="govuk-hint">
+          Do not include personal or financial information, like your National Insurance number or credit card details.
```

Comment:
@theKHutDeveloper - would be worth asking Leigh to provide some content for this as this hint is pretty generic/not relevant to our users.

Reply:
Changed hint text following your advice

<img width="672" alt="Screen Shot 2019-06-28 at 10 37 17" src="https://user-images.githubusercontent.com/10685841/60333641-76908080-9991-11e9-9cf3-25eb2a6f5452.png">